_internal/DeerLister.php

```diff
@@ -123,10 +123,28 @@ private function readDirectory(string $directory): array|false
         return $files;
     }
 
-    private function isHidden(string $path): bool
+    /**
+     * Returns if a file/folder should be displayed or not
+     *
+     * @param string $path Path to the file/folder
+     * @param mixed $config config.yaml file
+     * @param bool $ignoreHide Do we only consider forbidden files (true) or also hidden ones (false)
+    */
+    private function isHidden(string $path, mixed $config, bool $ignoreHide): bool
     {
         // files to hide, could array_merge with hidden files from a config
```

Comment:
Outdated comment

Reply:
What do you mean?

Reply:
Line 128, that's still from when we had an hardcoded array but that's no longer that case.